.github/workflows/package.yml

```
@@ -0,0 +1,40 @@
+
+  pull_request:
+      branches: [ "develop" ]
+      paths-ignore:
+      - target/**
+     
+env:
+  ENV_VAR: test
+
+jobs:
+  Package-test:
+      runs-on: ubuntu-latest
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 4 but found 6

(indentation)